From: Dmitry Shachnev <mitya57@debian.org>
Date: Sat, 30 Oct 2021 20:54:36 +0300
Subject: Remove code that relied on setuptools calling 2to3

Setuptools no longer runs that, we run 2to3 from debian/rules.
---
 setup.cfg    |  1 -
 setup.py     | 12 ------------
 setup3lib.py |  2 +-
 3 files changed, 1 insertion(+), 14 deletions(-)

diff --git a/setup.cfg b/setup.cfg
index 9e927a5..308e4de 100644
--- a/setup.cfg
+++ b/setup.cfg
@@ -2,7 +2,6 @@
 with-doctest = 1
 doctest-extension = .rst
 doctest-fixtures = _fixtures
-py3where = build/tests
 
 [bdist_rpm]
 doc_files = README.txt
diff --git a/setup.py b/setup.py
index 29e29a8..9499468 100644
--- a/setup.py
+++ b/setup.py
@@ -6,17 +6,6 @@ py_vers_tag = '-%s.%s' % sys.version_info[:2]
 
 test_dirs = ['functional_tests', 'unit_tests', os.path.join('doc','doc_tests'), 'nose']
 
-if sys.version_info >= (3,):
-    import setuptools
-
-    extra = {'use_2to3': True,
-             'test_dirs': test_dirs,
-             'test_build_dir': 'build/tests',
-             'pyversion_patching': True,
-             }
-else:
-    extra = {}
-
 try:
     from setup3lib import setup
     from setuptools import find_packages
@@ -36,7 +25,6 @@ try:
     if sys.version_info.major == 2:
         addl_args['entry_points']['console_scripts'].append(
             'nosetests%s = nose:run_exit' % py_vers_tag)
-    addl_args.update(extra)
 
     # This is required by multiprocess plugin; on Windows, if
     # the launch script is not import-safe, spawned processes
diff --git a/setup3lib.py b/setup3lib.py
index 27bdb93..51164b8 100644
--- a/setup3lib.py
+++ b/setup3lib.py
@@ -3,7 +3,7 @@ from setuptools import setup as _setup
 
 py3_args = ['use_2to3', 'convert_2to3_doctests', 'use_2to3_fixers', 'test_dirs', 'test_build_dir', 'doctest_exts', 'pyversion_patching']
 
-if sys.version_info < (3,):
+if True:
     # Remove any Python-3.x-only arguments (so they don't generate complaints
     # from 2.x setuptools) and then just pass through to the regular setup
     # routine.
